Presscotts_Mortar_Calculator_(no_envirement)_V0.2.py

```diff
@@ -246,120 +255,121 @@ def main():
     range2Upper = 'a'
     range2Lower = 'a'
     ''' l0A goes with range0B and vice versea '''
-    
-    if distance > 3100: 
-        print( "ERROR: OUT OF RANGE (Target Too Far)")
+
+    if distance > 3100:
+        print("ERROR: OUT OF RANGE (Target Too Far)")
         main()
     if distance < 50:
-        print( "ERROR: OUT OF RANGE (Target Too close)")
+        print("ERROR: OUT OF RANGE (Target Too close)")
         main()
-        
-    
     '''finding the valid ranges'''
-   
     '''charge 0'''
     for i in range(len(l0)):
         if distance < l0[i]:
             allNear0.append(l0[i])
         if distance > l0[i]:
-            allFar0.append(l0[i])    
-    
+            allFar0.append(l0[i])
+
     if len(allFar0) == 0 or len(allNear0) == 0:
         #print("No valid Range in Charge 0")
         charge0Truth = 0
     else:
-        allFar0.sort(reverse=True)   
+        allFar0.sort(reverse=True)
         allNear0.sort(reverse=False)
         range0Upper = allNear0[0]
         range0Lower = allFar0[0]
         charge0Truth = 1
-  
-    '''charge 1'''   
+    '''charge 1'''
     for i in range(len(l1)):
         if distance < l1[i]:
             allNear1.append(l1[i])
         if distance > l1[i]:
-            allFar1.append(l1[i])    
-    
+            allFar1.append(l1[i])
+
     if len(allFar1) == 0 or len(allNear1) == 0:
         #print("No valid Range in Charge 1")
         charge1Truth = 0

```

Comment:
These `Truth` variables seem to be boolean. While integers do implicitly convert to booleans, prefer using the actual boolean type:
```
charge1 = False
...
if (charge1):
  // doStuff
```